clients/cli/build.gradle.kts

```diff
@@ -67,3 +67,14 @@ tasks.jar {
   from(dependencies)
   duplicatesStrategy = DuplicatesStrategy.EXCLUDE
 }
+
+tasks.test {
+  val skipITs = project.hasProperty("skipITs")
+  if (skipITs) {
+    // Exclude integration tests
+    exclude("**/integration/test/**")
+  } else {
+    dependsOn(tasks.jar)
+    dependsOn(":catalogs:catalog-jdbc-postgresql:jar", ":catalogs:catalog-jdbc-postgresql:runtimeJars")
```

Comment:
Why does it only depends on `jdbc-postgresql`,do we need to add `jdbc-mysql` here?

Reply:
We only use `catalog-jdbc-postgresql` to run the integration test.
We didn't need support from many different catalogs to test.